### ISSUE TITLE
Reorder CC license options to CC, no CC, CC 0, public domain in submission form

### ISSUE THREAD

**Issue by alawvt** [VTUL/vtechworks#675]
Reorder CC license options to CC, no CC, CC 0, public domain in submission form

Currently we have, https://github.com/VTUL/vtechworks/blob/46faa5183036b5625c43c3bea7bf18f058d3cc77/dspace/modules/xmlui/src/main/webapp/i18n/messages.xml#L976

```
<message key="xmlui.Submission.submit.CCLicenseStep.info1"><p>Let people know whether and how they may use this work. Learn more at <strong><a href="http://creativecommons.org" title="Creative Commons" target="_blank">Creative Commons.</a></strong></p>
<h4>Public Domain - no known copyright</h4>
<p>The Public Domain mark by Creative Commons indicates that neither you nor anyone else currently has rights to the work. Common reasons for a work to be in the public domain include because its copyright has expired, because it was created by a government body, or because it is factual and therefore does not meet the copyright standard for original creative expression. The Public Domain mark is most often appropriate for texts and images published before the early 20th century, government documents, datasets, graphs, and charts.</p>
<h4>CC0 - no rights reserved</h4>
<p>CC0 ("Creative Commons Zero") enables you to waive your rights in your works and thereby place them as completely as possible in the public domain, so that others may freely build upon, enhance, and reuse your works for any purposes without restriction under copyright or database law.</p>
<h4>Creative Commons - some rights reserved</h4>
<p>Creative Commons licenses allow you to retain some rights: this option will let you decide and declare whether to allow or refuse permission for commercial uses of your work, whether to allow or refuse permission to modify your work, or whether to allow modifications of your work only on the condition that modifiers also allow others to modify the new work ("ShareAlike").</p>
<h4>No Creative Commons license</h4>
<p>You need not specify a Creative Commons license or mark for the work you are uploading, but in that case, people who find the content online will not know whether or how they are permitted to use, share, redistribute, remix, tweak, and build upon your work.</p></message>
```
Replace with
```
<message key="xmlui.Submission.submit.CCLicenseStep.info1"><p>Let people know whether and how they may use this work. Learn more at <strong><a href="http://creativecommons.org" title="Creative Commons" target="_blank">Creative Commons.</a></strong></p>
<h4>Creative Commons - some rights reserved</h4>
<p>Creative Commons licenses allow you to retain some rights: this option will let you decide and declare whether to allow or refuse permission for commercial uses of your work, whether to allow or refuse permission to modify your work, or whether to allow modifications of your work only on the condition that modifiers also allow others to modify the new work ("ShareAlike").</p>
<h4>No Creative Commons license</h4>
<p>You need not specify a Creative Commons license or mark for the work you are uploading, but in that case, people who find the content online will not know whether or how they are permitted to use, share, redistribute, remix, tweak, and build upon your work.</p>
<h4>CC0 - no rights reserved</h4>
<p>CC0 ("Creative Commons Zero") enables you to waive your rights in your works and thereby place them as completely as possible in the public domain, so that others may freely build upon, enhance, and reuse your works for any purposes without restriction under copyright or database law.</p>
<h4>Public Domain - no known copyright</h4>
<p>The Public Domain mark by Creative Commons indicates that neither you nor anyone else currently has rights to the work. Common reasons for a work to be in the public domain include because its copyright has expired, because it was created by a government body, or because it is factual and therefore does not meet the copyright standard for original creative expression. The Public Domain mark is most often appropriate for texts and images published before the early 20th century, government documents, datasets, graphs, and charts.</p>
</message>
```

**Comment by alawvt** [comment]
In researching this issue, I reported [Creative Commons step: Public Domain and CC0 choice have same result](https://jira.lyrasis.org/browse/DS-4406).

**Comment by alawvt** [comment]
Resolved by #677.
